src/Xamarin.Android.Build.Tasks/Utilities/AndroidResource.cs

```diff
@@ -202,6 +202,8 @@ private static bool TryFixResAuto (XAttribute attr, Dictionary<string, string> a
 		{
 			if (attr.Name.Namespace != res_auto)
 				return false;
+			if (!attr.Value.StartsWith ("@")) // only lowercase if we reference another resource.
```

Comment:
The only think we should probably do is:
```suggestion
			if (!attr.Value.StartsWith ("@", StringComparison.Ordinal)) // only lowercase if we reference another resource.
```